src/dune_engine/scheduler.ml

```diff
@@ -365,20 +359,20 @@ end = struct
         Option.map (Queue.pop q.jobs_completed) ~f:(fun (job, proc_info) ->
             q.pending_jobs <- q.pending_jobs - 1;
             assert (q.pending_jobs >= 0);
-            Job_completed (job, proc_info))
+            Fill (Fill (job.ivar, proc_info)))
```

Comment:
This is a little hard to read. How about calling the new event `Fiber_ivar_fill` instead?

Reply:
I called it `Fiber_fill_ivar`.